src/app/icon/icon.module.ts

```diff
@@ -0,0 +1,17 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { MzIconMdiDirective } from './icon-mdi.directive';
+import { MzIconDirective } from './icon.directive';
+
+@NgModule({
+  declarations: [
+    MzIconMdiDirective,
+    MzIconDirective,
+  ],
+  exports: [
+    MzIconMdiDirective,
+    MzIconDirective,
+  ],
+})
+export class MzIconModule { }
```

Comment:
Should we do 2 modules for icons or would it be too confusing?
- MzIconModule
- MzIconMdiModule

Reply:
You are right we should have one module for each one.